### PR TITLE
OTLP: Move policy name to attributes

### PR DESCRIPTION
**Pull request by leoparente**
- Move pktvisor policy_name to attributes.
- No name is set for the scope anymore (i'm not sure if it is required)

- It requires validation! cc @manrodrigues

### CODE REVIEW

File: src/CoreServer.cpp [hunk 1]
```diff
@@ -448,7 +448,9 @@ void CoreServer::_setup_routes(const PrometheusConfig &prom_config)
                 try {
                     auto [policy, lock] = _registry->policy_manager()->module_get_locked(p_mname);
                     auto scope = resource.add_scope_metrics();
-                    scope->mutable_scope()->set_name(p_mname);
+                    auto attr = scope->mutable_scope()->add_attributes();
```
Comment (lpegoraro) [inline src/CoreServer.cpp:452]: nice!
with this the scope name should have something like "pktvisorreceiver"
right @weyrick ?

Comment (weyrick) [reply src/CoreServer.cpp:452]: @leoparente can we make `scope->mutable_scope()->set_name();` "pktvisor/POLICYNAME" please?

Comment (leoparente) [reply src/CoreServer.cpp:452]: sure

Comment (leoparente) [reply src/CoreServer.cpp:452]: done

**Review by weyrick — APPROVED** [review]

**Pull request merged**